docs/development/journalist_api.rst

```diff
@@ -0,0 +1,414 @@
+Journalist Interface API
+========================
+
+This document describes the endpoints for SecureDrop's Journalist Interface
+API.
+
+Versioning
+~~~~~~~~~~
+
+The API is versioned and we are currently using version 1. This is set via the
+base URL, which is:
+
+.. code:: sh
+
+  /api/v1/
+
+Content Type
+~~~~~~~~~~~~
+
+Clients should send the following headers:
+
+.. code:: sh
+
+  'Accept': 'application/json',
+  'Content-Type': 'application/json'
+
+Authentication
+~~~~~~~~~~~~~~
+
+``POST /api/v1/token`` to get a token with the username, password, and 2FA
+token in the request body:
+
+.. code:: sh
+
+  {
+  	"username": "journalist",
+  	"password": "mypasswordgoeshere",
+  	"one_time_code": "123456"
+  }
+
+Thereafter in order to authenticate to protected endpoints, send the token in
+HTTP Authorization header:
+
+.. code:: sh
+
+  Authorization: Token yourtokengoeshere
```

Comment:
I think `Bearer <token>` is more canonical. The Rust http lib has native support for that, and this is usually the value I've seen in the workplace.